### PR TITLE
Add pre-built cohort into adult census notebook

### PR DESCRIPTION
**Pull request by gaugup**
## Description

Adding SDK cohorts to adult census notebook.

## Areas changed

npm packages changed:

- [ ] responsibleai/causality
- [ ] responsibleai/core-ui
- [ ] responsibleai/counterfactuals
- [ ] responsibleai/dataset-explorer
- [ ] responsibleai/fairness
- [ ] responsibleai/interpret
- [ ] responsibleai/localization
- [ ] responsibleai/mlchartlib
- [ ] responsibleai/model-assessment

Python packages changed:

- [ ] raiwidgets
- [ ] responsibleai
- [ ] erroranalysis
- [ ] rai_core_flask

## Tests

- [x] No new tests required.
- [ ] New tests for the added feature are part of this PR.
- [ ] I validated the changes manually.

## Documentation:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb [hunk 1]
```diff
@@ -276,7 +346,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "ResponsibleAIDashboard(rai_insights)"
+    "widget = ResponsibleAIDashboard(rai_insights, cohort_list=cohort_list)"
```
Comment (imatiach-msft) [inline notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb:349]: why does "widget =" need to be assigned here?

**Review by imatiach-msft — APPROVED** [review]

File: notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb (hunk 1 above)
Comment (gaugup) [reply notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb]: Removed it.

File: notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb [hunk 2]
```diff
@@ -257,6 +257,80 @@
...
+    "cohort_filter_continuous_2 = CohortFilter(\n",
+    "    method=CohortFilterMethods.METHOD_GREATER,\n",
+    "    arg=[40],\n",
+    "    column='hours-per-week')\n",
+    "\n",
+    "user_cohort_continuous = Cohort(name='Cohort Continuous')\n",
+    "user_cohort_continuous.add_cohort_filter(cohort_filter_continuous_1)\n",
+    "user_cohort_continuous.add_cohort_filter(cohort_filter_continuous_2)\n",
+    "\n",
+    "# Cohort on categorical feature in the dataset\n",
```
Comment (romanlutz) [inline notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb:291]: Did you mean "features" (plural)? In any case, it's not obvious to me why it's named like this. How about naming them after the filters you put, e.g., age between 40 and 65?

File: apps/widget-e2e/src/describer/modelAssessment/modelAssessmentDatasets.ts [hunk 3]
```diff
@@ -53,6 +53,14 @@ const modelAssessmentDatasets = {
       "capital-loss"
     ],
     modelStatisticsData: {
+      cohortDropDownValues: [
+        "All data",
+        "Cohort Continuous",
```
[the remark this answers is not in the document]
Comment (gaugup) [reply apps/widget-e2e/src/describer/modelAssessment/modelAssessmentDatasets.ts]: Agreed. These should not be the names for a user facing notebook. I updated these to meaningful names for "Cohort Continuous" and "Cohort Categorical".

File: notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb [hunk 4]
```diff
@@ -154,7 +154,7 @@
     "\n",
     "\n",
     "# Take 500 samples from the test data\n",
-    "test_data_sample = test_data.sample(n=500, random_state=5)"
+    "test_data_sample = test_data.sample(n=50, random_state=5)"
```
[the remark this answers is not in the document]
Comment (gaugup) [reply notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb:152]: Updated.

File: notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb (hunk 2 above)
Comment (gaugup) [reply notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb]: Updated. Please check.

**Review by romanlutz — APPROVED** [review]

**Pull request merged**